docs/src/pages/[platform]/connected-components/authenticator/quick-start-add.android.mdx

```diff
@@ -0,0 +1,27 @@
+The quickest way to get started is by wrapping your App component (written in Jetpack Compose) with the Authenticator widget.
```

Comment:
Somewhere in the quick start instructions, can we add a note about setting up Amplify Auth and link to the Amplify Android Auth documentation?